src/cmd/usdConverter.cc

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <iostream>
+
+#include <pxr/usd/usd/stage.h>
+
+#include "sdf/sdf.hh"
+#include "sdf_usd_parser/world.hh"
+
+int main(int argc, const char* argv[])
+{
+  if (argc != 3)
+  {
+    std::cerr << "Usage: " << argv[0] << " <sdf-path> <usd-path>\n";
+    return -1;
+  }
+
+  sdf::Root root;
+  auto errors = root.Load(argv[1]);
+  if (!errors.empty())
+  {
+    std::cerr << "Errors encountered:\n";
+    for (const auto &e : errors)
+      std::cout << e << "\n";
+    return -2;
+  }
+
+  // only support SDF files with exactly 1 world for now
+  if (root.WorldCount() != 1u)
+  {
+    std::cerr << argv[1] << " does not have exactly 1 world\n";
+    return -3;
+  }
```

Comment:
Do you have an idea on how support for multi world sdformat look like?

I'm thinking one approach would be to create a root stage containing sub layers, a downside of this is that afaik the sub layers must be separate files, and that usd always composes the sublayers into a single scene.

Do you know how multi world sdformat is handled in ignition/gazebo? Or they composed into a single scene like usd, or is only one world is rendered at a time?